sdk/core/azure-core/tests/tracing_common.py

```diff
@@ -64,25 +65,24 @@ class MockExporter(Exporter):
     def __init__(self):
         self.root = None
         self._all_nodes = []
+        self.parent_dict = {}
 
     def export(self, span_datas):
         # type: (List[SpanData]) -> None
         sp = span_datas[0]  # type: SpanData
         node = Node(sp)
         if not node.span_data.parent_span_id:
             self.root = node
+        parent_span_id = node.span_data.parent_span_id
+        if parent_span_id not in self.parent_dict:
+            self.parent_dict[parent_span_id] = []
+        self.parent_dict[parent_span_id].append(node)
```

Comment:
could use [`defaultdict(list)`](https://docs.python.org/2/library/collections.html#collections.defaultdict) to avoid the explicit boilerplate conditional

Reply:
done